README.md

```diff
@@ -27,11 +27,31 @@ Log into database to check whether the tables are created and populated with som
 ```
 docker compose exec db psql -h db -U postgres
 ```
+
+Start control service on a new tab, to later post a Sprinkler json workflow
+```
+go run . service control --config .sprinkler.yaml 
+```
+
+Post to `http://localhost:8080/v1/workflow` a Sprinkler workflow, example payload from [ExampleWorkflow.scala](https://github.com/salesforce/spade/blob/main/spade-examples/src/main/scala/com/salesforce/mce/spade/examples/ExampleWorkflow.scala)
+```
+{
+    "name": "test-workflow",
+    "artifact": "s3://sprinkler-salesforce-bucket/jars/test/spade-example.jar",
+    "command":  "[\"java\", \"-cp\", \"spade-example.jar\", \"com.salesforce.mce.spade.examples.ExampleWorkflow\", \"generate\", \"--compact\"]",
+    "every": "60 minute",
+    "nextRuntime": "2023-03-22T21:00:00Z",
+    "backfill": false,
+    "owner": "mc-kernel",
```

Comment:
this is not implemented yet, but it's suppose to be a valid email address/sns-arn

Reply:
something like this?  `arn:aws:sns:us-east-2:444455556666:MyTopic`